go/api/command_options.go

```diff
@@ -355,3 +355,41 @@ func (opts *RestoreOptions) toArgs() ([]string, error) {
 	}
 	return args, err
 }
+
+// Optional arguments to Copy(source string, destination string, option *CopyOptions)
+//
+// [valkey.io]: https://valkey.io/commands/Copy/
+type CopyOptions struct {
+	// The REPLACE option removes the destination key before copying the value to it.
+	replace string
```

Comment:
```suggestion
	replace bool
```
REPLACE is a constant keyword in the command. The above option can be bool.